documentation/jetty-documentation/src/main/java/org/eclipse/jetty/docs/programming/server/http/HTTPServerDocs.java

```diff
@@ -999,22 +1006,16 @@ public void defaultHandler() throws Exception
     {
         // tag::defaultHandler[]
         Server server = new Server();
+        server.setDefaultHandler(new DefaultHandler());
```

Comment:
Is this explicit set needed?

Reply:
It's not needed, but it is used for the documentation about how to set a `DefaultHandler`.  I'll add some configuration to the example to make it more meaningful.